ontology/yaml/resources/LIGHTING/entity_types/GENERALTYPES.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+LT:
+  description: "Luminaire, lighting fixture."
+  implements:
+  - EQUIPMENT
+  uses:
+  - dimmer_percentage_command # in % indicating brightness
+
+LPS:
+  description: "Lighting photocell sensor."
+  implements:
+  - MTS	
+
+LMS:
+  description: "Lighting motion sensor."
+  implements:
+  - MTS	
+
+LTGW:
```

Comment:
If this is a controller, its probably not appropriate in the lighting namespace. If you really want to model the gateway, it would probably be best in its own 'NETWORKING' namespace, or something similar. If we look to HVAC for precedent, there is no modeling of individual controllers or gateways; those aren't the target for analysis.

Reply:
Modelling the lighting gateway is useful so that we can attribute fields such as electrical power consumption and more importantly high level scenes and groups information. It is a lighting networking device, not an IP networking device, so I still lean towards making it a lighting device. Energy information is useful to tackle use cases like reducing ghost/parasitic power loads, which in modern buildings can easily consume 1/3 and more of the total energy consumption of a building.